src/csharp/Intel/Iced/Intel/Assembler/Assembler.cs

```diff
@@ -167,15 +167,22 @@ public partial class Assembler {
 		/// Add an instruction directly to the flow of instructions.
 		/// </summary>
 		/// <param name="instruction"></param>
-		/// <param name="flags">Operand flags passed.</param>
-		public void AddInstruction(Instruction instruction, AssemblerOperandFlags flags = AssemblerOperandFlags.None) {
+		public void AddInstruction(Instruction instruction) {
```

Comment:
Is this method needed and does it need to be public? All instructions, except eg. old 286 loadall instructions, are supported by the `Assembler`.

Reply:
What about `Instructions` and `CurrentLabel`, when would the user need to use those properties?

Reply:
They are useful when you can to copy from an existing sets of instructions (Copy from another Assembler instance or could whatever list of instructions).